media-gfx/openexr_viewers/openexr_viewers-2.3.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools flag-o-matic
+
+DESCRIPTION="OpenEXR Viewers"
+HOMEPAGE="http://openexr.com/"
+SRC_URI="https://github.com/openexr/openexr/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd"
+IUSE="cg"
+
+RDEPEND="~media-libs/ilmbase-${PV}:=
+	~media-libs/openexr-${PV}:=
+	virtual/opengl
+	x11-libs/fltk:1[opengl]
+	cg? ( media-gfx/nvidia-cg-toolkit )"
+#	>=media-libs/ctl-1.5.2:=
```

Comment:
Why is it commented out?

Reply:
I commented that out too see if it would work and forgot to remove it. 2.3.0 dropped this as a dependency!